tests/test_cleed_mode.c

```
@@ -0,0 +1,76 @@
+    CLEED_TEST_ASSERT(cleed_parse_mode_value("YES") == CLEED_MODE_SYM);
+    CLEED_TEST_ASSERT(cleed_parse_mode_value("true") == CLEED_MODE_SYM);
+    CLEED_TEST_ASSERT(cleed_parse_mode_value("1") == CLEED_MODE_SYM);
+    CLEED_TEST_ASSERT(cleed_parse_mode_value("no") == CLEED_MODE_NSYM);
+    CLEED_TEST_ASSERT(cleed_parse_mode_value("FALSE") == CLEED_MODE_NSYM);
+    CLEED_TEST_ASSERT(cleed_parse_mode_value("0") == CLEED_MODE_NSYM);
+    CLEED_TEST_ASSERT(cleed_parse_mode_value("maybe") == CLEED_MODE_INVALID);
+
+    {
+        char path[256];
+        int has_symmetry = 0;
+        CLEED_TEST_ASSERT(write_temp_file("sr: 3 0.0 0.0\n", path, sizeof(path)) == 0);
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)